proxy/http2/Http2ConnectionState.cc

```diff
@@ -970,8 +970,8 @@ Http2ConnectionState::main_event_handler(int event, void *edata)
       const char *client_ip = ats_ip_ntop(ua_session->get_client_addr(), ipb, sizeof(ipb));
       if (error.cls == Http2ErrorClass::HTTP2_ERROR_CLASS_CONNECTION) {
         if (error.msg) {
-          Error("HTTP/2 connection error client_ip=%s session_id=%" PRId64 " stream_id=%u %s", client_ip,
-                ua_session->connection_id(), stream_id, error.msg);
+          Error("HTTP/2 connection error code=%d client_ip=%s session_id=%" PRId64 " stream_id=%u %s", static_cast<int>(error.code),
```

Comment:
Looks okay.
Is it better to convert the code to string to log, so user don't need to search the source to find out what the actual meaning?

Reply:
Well, I'm ok with using error names instead of codes. I'm not too worried about the log size but a line might get a bit too long, and making consistent length value might be easy to read (the maximum error code is 0x0d if we use hex).

Reply:
Now it prints error codes in hex decimal. Let me know if you still want it to be a string error name.